lib/h3core.js

```diff
@@ -65,6 +65,17 @@ const SZ_COORDIJ = H3.sizeOfCoordIJ();
 
 // ----------------------------------------------------------------------------
 // Unit constants
+
+/**
+ * Length/Area units
+ * @static
+ * @typedef UNITS
+ * @type {Object}
+ * @property {string} m
+ * @property {string} km
+ * @property {string} m2
+ * @property {string} km2
```

Comment:
I wish there was a way to read this directly from the code, but I guess that's a limitation of the JSDoc-driven type defs 🤷‍♂️